app/views/ops/_schedule_form_timer.html.haml

```diff
@@ -43,6 +43,7 @@
                    :name             => "start_date",
                    "ng-model"        => "scheduleModel.start_date",
                    "datepicker-init" => "",
+                   "miq-calendar"    => true,
```

Comment:
We should merge `datepicker-init` and `miq-calendar` but until then, probably shouldn't use both at the same time.. or rather, we just need the parser & formatter combo from miqCalendar here, so maybe we can split miqCalendar into 2 .. because it also watches ngModel and triggers an update, which could potentially happen *before* datepicker-init has a chance to initialize the component (because it does so only after it changes to something other than undefined. Oh.. datepickerInit reinitializes it on every update, instead of calling update... 

Uh.. can this wait until we :boom: :bomb: datepickerInit and move the missing functionality to miqCalendar? :)